test/apiv2/20-containers.at

```diff
@@ -676,12 +687,20 @@ if root; then
   echo '{"Memory":{"Limit":500000}, "CPU":{"Shares":123}}' >${TMPD}/update.json
   t POST libpod/containers/updateCtr/update ${TMPD}/update.json 201
 
+  # 002 is the byte length
+  result=$'\001\0025'
+  cgroupPath=/sys/fs/cgroup/cpu.weight
```

Comment:
My eye in reverse order finds it to read difficult. Would you mind defining `cgroupPath` first, and then the second one, which should perhaps be named `expect` instead of `result`?